extension/js/common/core/crypto/pgp/openpgp-key.ts

```diff
@@ -27,21 +34,32 @@ export class OpenPGPKey {
   };
 
   public static parseMany = async (text: string): Promise<Key[]> => {
-    const result = await opgp.key.readArmored(text);
+    const resultKeys = await opgp.readKeys({ armoredKeys: text });
+    /* todo: test exception and re-wrap?
     if (result.err) {
       throw new Error('Cannot parse OpenPGP key: ' + result.err + ' for: ' + text);
     }
-    const keys = [];
-    for (const key of result.keys) {
+    */
+    const keys: Key[] = [];
+    for (const key of resultKeys) {
       await OpenPGPKey.validateAllDecryptedPackets(key);
       keys.push(await OpenPGPKey.convertExternalLibraryObjToKey(key));
     }
     return keys;
   };
 
-  public static validateAllDecryptedPackets = async (key: OpenPGP.key.Key): Promise<void> => {
+  // remove crypto-library objects (useful when sending the object to/from background)
+  public static pack = (key: Key) => {
+    const keyWithPrivateFields = key as unknown as KeyWithPrivateFields;
+    if (typeof keyWithPrivateFields.internal !== 'string') {
+      keyWithPrivateFields.internal = keyWithPrivateFields.internal.armor();
+      keyWithPrivateFields.rawKey = undefined;
+    }
+  };
```

Comment:
Simplicity looks good.

Would it be faster to send the key in binary representation (unarmored openpgp key)? Armoring/de-armoring is oddly expensive, about 10ms per key. Could be separate PR.

Reply:
https://github.com/FlowCrypt/flowcrypt-browser/issues/4904

Reply:
I added this into this PR